### PR TITLE
Update the classes doc

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
See https://github.com/mdn/content/pull/17054

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/classes/index.html [hunk 1]
```diff
@@ -225,6 +225,8 @@ <h4 id="公有字段声明">公有字段声明</h4>
 
 <p>通过预先声明字段，类定义变得更加自我记录，并且字段始终存在。</p>
 
+<p>在声明字段时，我们不需要像<code>let</code>，<code>const</code>和<code>var<code>这样的关键字。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/classes/index.html:228]:
```suggestion
<p>在声明字段时，我们不需要像 <code>let</code>、<code>const</code> 和 <code>var<code> 这样的关键字。</p>
```

**Review by yin1999 — COMMENTED** [review]
In zh-CN, we use

File: files/zh-cn/web/javascript/reference/classes/index.html [hunk 2]
```diff
@@ -225,6 +225,8 @@ <h4 id="公有字段声明">公有字段声明</h4>
 
 <p>通过预先声明字段，类定义变得更加自我记录，并且字段始终存在。</p>
 
+<p>在声明字段时，我们不需要像 <code>let</code>、<code>const</code> 和 <code>var<code> 这样的关键字。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/classes/index.html:228]:
```suggestion
<p>在声明字段时，我们不需要像 <code>let</code>、<code>const</code> 和 <code>var</code> 这样的关键字。</p>
```

**Comment by awxiaoxian2020** [comment]
> In zh-CN, we use

LGTM! Thanks!

**Pull request merged**